rfd/0058-oci-public-images.md

```diff
@@ -0,0 +1,208 @@
+
+#### **Third-Party Registries**
+While both third-party registries presented below would be less overhead to manage. They both involve losing control over the distribution of our software, which is a key point that this RFD and other RFDs encourage. 
+
+However, in order to address the issue of name squatting, and to provide customers a potentially improved experience, an RFD on [Artifact Mirroring](https://github.com/gravitational/teleport/commit/2262efbb25463ccc135553d43293f6d8aee22ba2) will be created. 
+
+**Dockerhub**
+
+Docker Hub supports bringing your own [SSO provider](https://docs.docker.com/single-sign-on/) and [vulnerability scanning](https://docs.docker.com/docker-hub/vulnerability-scanning/). This choice would produce the strongest user experience as images could be named `gravitational/teleport` or even just `teleport` if Docker allowed. 
+
+**Amazon ECR Public**
+
+Amazon ECR Public supports [image scanning](https://docs.aws.amazon.com/AmazonECR/latest/userguide/image-scanning.html) and would leverage our existing Okta SSO infrastructure. Additionally, repositories would be managed using Terraform IaC. However, Amazon ECR Public doesn't have support for custom domains. This means images would be pulled using `public.ecr.aws/gravitational/teleport`.
+
+### **Implementation**
+TODO: Include in-depth step by step guide on how the above solution will be created and migrated
+
+* Multi step process. AWS ECR Infrastructure in the `cloud-terraform` repository
+* Push to Quay and Harbor
+* Replicate existing images from quay.io to ECR
+* update documentation to new location 
+* retire quay repository
+
+## References
+\[1\] - https://access.redhat.com/articles/5925591
+\[2\] - https://goharbor.io/
+\[3\] - https://goharbor.io/docs/2.4.0/administration/configure-authentication/oidc-auth/
+
+[1]: https://access.redhat.com/articles/5925591
+[2]: https://goharbor.io/
+[3]: https://goharbor.io/docs/2.4.0/administration/configure-authentication/oidc-auth/
+
+## Brain Dump
+
+* Is discoverability of the images and tags in scope for this RFD? Should users just assume that for each valid release there is a valid tag? Must ensure that release pipeline succeeds or a core release member should be alerted to fix any issues. 
+* Make sure to give reference to precedence of existing RFDS such as:
+    * [Terraform Registry](https://github.com/gravitational/teleport-plugins/blob/master/rfd/0002-custom-terraform-registry.md)
+    * [Release Asset Management](https://github.com/gravitational/cloud/blob/master/rfd/0004-Release-Asset-Management.md)
+    * [Package Distribution](https://github.com/gravitational/teleport/pull/10746)
+    * [Artifact Storage Standards](https://github.com/gravitational/cloud/blob/master/rfd/0017-artifact-storage-standards.md)
+
+### Out of scope 
```

Comment:
I'd move all this up to the scope section and add a sentance of detail to each point.